string timestamp
string cmd         #任务
string subcmd      #交通信号灯、控制状态
pathpose[] pose